src/content/en/fundamentals/security/prevent-mixed-content/what-is-mixed-content.md

```diff
@@ -14,18 +14,19 @@ description: Mixed occurs when initial HTML is loaded over a secure HTTPS connec
 connection, but other resources (such as images, videos, stylesheets, scripts)
 are loaded over an insecure HTTP connection. This is called mixed content
 because both HTTP and HTTPS content are being loaded to display the same page,
-and the initial request was secure over HTTPS. Modern browsers display warnings
-about this type of content to indicate to the user that this page contains
-insecure resources.
+and the initial request was secure over HTTPS. 
+
+Modern browsers may block this content, or will display warnings about this type of content to indicate to the user that this page contains insecure resources. Browsers that block mixed content may first attempt to "upgrade" the connection to this content from HTTP to HTTPS.
 
 ### TL;DR {: .hide-from-toc }
 
 * HTTPS is important to protect both your site and your users from attack.
 * Mixed content degrades the security and user experience of your HTTPS site.
+* Browsers may block mixed content altogether over time, in order to protect your site's users.
```

Comment:
It seemed significant enough to add it to the top TL;DR bullets, even though so far it's just one browser that's announced this change. Again, not sure how we want to phrase single-browser initiatives.

Reply:
What about something like: "Different browsers block different types of mixed content, and Chrome will soon block all mixed content, so it's best to avoid mixed content altogether." (with a link to the blog post)